src/math/numbers.c

```diff
@@ -6,25 +6,59 @@
 //         Liam Girdwood <liam.r.girdwood@linux.intel.com>
 //         Keyon Jie <yang.jie@linux.intel.com>
 
-/* Euclidean algorithm for greatest common denominator from
- * pseudocode in
- * https://en.wikipedia.org/wiki/Euclidean_algorithm#Implementations
- */
-
 #include <sof/audio/format.h>
 #include <sof/math/numbers.h>
 #include <stdint.h>
 
+/* Binary GCD algorithm
+ * https://en.wikipedia.org/wiki/Binary_GCD_algorithm
+ * This function returns the greatest common divisor of two numbers
```

Comment:
So, what I meant to say is to move the first two lines from here:
```
/* Binary GCD algorithm
 * https://en.wikipedia.org/wiki/Binary_GCD_algorithm
```
in the place of the previous comments:

```
/* Euclidean algorithm for greatest common denominator from
 * pseudocode in
 * https://en.wikipedia.org/wiki/Euclidean_algorithm#Implementations
 */
```
so the final diff will look something like this:
```
-/* Euclidean algorithm for greatest common denominator from
- * pseudocode in
- * https://en.wikipedia.org/wiki/Euclidean_algorithm#Implementations
- */
+/* Binary GCD algorithm
 + * https://en.wikipedia.org/wiki/Binary_GCD_algorithm
 + */

#include <sof/audio/format.h>
#include <sof/math/numbers.h>
#include <stdint.h>

+ * This function returns the greatest common divisor of two numbers
+ ....
```